<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sudoku</class>
 <widget class="QMainWindow" name="Sudoku">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sudoku</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(154, 153, 150);</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QFrame" name="verticalFrame">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgba(187, 183, 183, 251);
	border-style: outset;
	border-width: 2px;
	border-radius: 20px;
	border-color: beige;
	padding: 6px;
}

QPushButton:hover {
    background-color: rgb(134, 129, 116);     
}
QPushButton:pressed {
    background-color: rgb(169, 189, 207);     
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>14</number>
       </property>
       <property name="bottomMargin">
        <number>13</number>
       </property>
       <item>
        <widget class="QPushButton" name="startGameBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>26</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Start Game</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="diffLevel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>26</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Difficulty Level</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="myScore">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>26</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>My Score</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitGame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>26</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Exit Game</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="sTitle">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Chilanka</family>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Sudoku</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Chilanka</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitGame</sender>
   <signal>clicked()</signal>
   <receiver>Sudoku</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
